audit/log4j-nessus-parser.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Usage:
+#
+#   ./$0 path-to-nessus.xml
+# 
+# This emits the paths, and hosts, for log4j findings, as CSV.  Getting 
+# the final output of responsible users is a few more steps.
+# 
+# At that point I suggest you take the output, and paste it into a google sheet
+# then copy all the instance guids, and be ready to paste them into a file
+# called `instance_guids` on a deigo cell:
+# 
+# Then, on a diego cell, run the following to echo as more CSV the
+# instance_guid and the app_guid.
+# 
+#   cat instance_guids | while read i; do
+#    if ( echo $i | grep -q -- - ); then
+#            jqcmd="jq '.LRPs[] | select (.instance_guid == \"$i\") | .process_guid'"
+#            echo -n "\"$i\", "
+#            cfdot cell-states | eval $jqcmd | cut -c1-36
+#    else
+#      echo "\"$i\", \"-\""
+#    fi
```

Comment:
does this get run once total, on one diego cell? if so, could we get it somewhere else that doesn't require shelling out?

Reply:
Prometheus and logsearch probably expose this (although that probably doesn't help directly).

Reply:
I think we need to use `cfdot` to get that mapping. Does cfdot work on other nodes besides diego?

Reply:
it seems plausible that we could configure it to run from a jumpbox, which seems like a slight improvement for the future